kfs-core/src/main/java/edu/arizona/kfs/pdp/batch/service/impl/ShippingInvoiceLoadServiceImpl.java

```diff
@@ -76,7 +80,7 @@ public boolean loadShippingInvoiceFile(String fileName) {
         }
         
         applyHeaderToInvoices(batchContents);
-        saveShippingData(batchContents);
+        saveShippingData(batchContents, fileName, SpringContext.getBean(DateTimeService.class).getCurrentTimestamp());

```

Comment:
Can we inject DateTimeService in the spring config instead of retrieving it from the SpringContext?